### DISCUSSION TITLE
Adding custom action button to .NET MAUI Shell TabBar

### DISCUSSION THREAD

**Discussion by giscus** [VladislavAntonyuk/vladislavantonyuk.github.io#113] [Announcements]
# Adding custom action button to .NET MAUI Shell TabBar

The article demonstrates how to customize .NET MAUI Shell TabBar for Android and iOS/MacCatalyst platforms, offering step-by-step guidance, code examples, and a link to the GitHub repository containing full examples for practical usage.

https://vladislavantonyuk.github.io/articles/Adding-custom-action-button-to-.NET-MAUI-Shell-TabBar/

**Comment by GoldenSpark1337** [comment]
Appreciate the work! But is there a way to make it a part of tabbar itself? As i understand correctly, it render's button and tabbar separately, so it makes things like Visibility of shell tabbar much harder. Maybe there is an easy way that i don't know
